src/utils.js

```diff
@@ -5,34 +5,42 @@ const chalk = require('chalk');
 const constants = require('./constants');
 const CONFIG_FILE = constants.files.configFile;
 
-module.exports.error = (message) => {
+let error = (message) => {
     console.error(`neu: ${chalk.bgRed.black('ERROR')} ${message}`);
 }
 
-module.exports.isNeutralinojsProject = () => {
+let isNeutralinojsProject = () => {
     return fs.existsSync(CONFIG_FILE);
 }
 
-module.exports.getFiglet = () => {
+let getFiglet = () => {
     return figlet.textSync('Neutralinojs');
 }
 
-module.exports.figlet = () => {
-    console.log(figlet.textSync('Neutralinojs'));
+let showArt = () => {
+    console.log(getFiglet());
 }
 
-module.exports.checkCurrentProject = () => {
+let checkCurrentProject = () => {
     if(!isNeutralinojsProject()) {
         error(`Unable to find ${CONFIG_FILE}. ` +
                     `Please check whether the current directory has a Neutralinojs project.`);
         process.exit(1);
     }
 }
 
-module.exports.log = (message) => {
+let log = (message) => {
     console.log(`neu: ${chalk.bgGreen.black('INFO')} ${message}`);
 }
 
-module.exports.warn = (message) => {
+let warn = (message) => {
     console.warn(`neu: ${chalk.bgYellow.black('WARNING')} ${message}`);
 }
+
+module.exports.error = error;
+module.exports.isNeutralinojsProject = isNeutralinojsProject;
+module.exports.getFiglet = getFiglet;
+module.exports.showArt = showArt;
+module.exports.checkCurrentProject = checkCurrentProject;
+module.exports.log = log;
+module.exports.warn = warn;
```

Comment:
Could you please use one `module.export` for this? All other things look great and ready to be merged :tada:

Eg:
```js
module.exports = {
    error,
    isNeutralinojsProject,
   ... 
 ... 
}
```

Reply:
Ahh, how did I miss that. Doing it right away.